.gitignore

```diff
@@ -9,6 +9,7 @@ db.json
 *.log
 public/
 .deploy*/
+dist
```

Comment:
Added this just to make sure I didn't commit this directory let me know if you want this removed

Reply:
We don't want to gitignore `dist` because it will eventually need to be added to the repo to provide direct repo installation from NPM.